notification/src/api/rest/notificationConfigEndpoint.ts

```diff
@@ -38,9 +36,9 @@ export class NotificationConfigEndpoint {
    */
   handleConfigSummaryRequest = async (req: express.Request, res: express.Response): Promise<void> => {
     const pipelineId = parseInt(req.params.id)
-    console.log(`Received request for configs with pipeline id ${pipelineId} from Host ${req.connection.remoteAddress}`)
+    console.log(`Received request for configs with pipeline id ${pipelineId}`)
```

Comment:
I think we should remove those logs, maybe replace them with logging middleware?

Reply:
I am going to remove them. For a logging/tracing middleware we should open a separate issue.